sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/descriptor/MetricDescriptor.java

```diff
@@ -46,21 +44,32 @@ public static MetricDescriptor create(View view, InstrumentDescriptor instrument
     String name = (view.getName() == null) ? instrument.getName() : view.getName();
     String description =
         (view.getDescription() == null) ? instrument.getDescription() : view.getDescription();
-    return new AutoValue_MetricDescriptor(
-        name, description, instrument.getUnit(), Optional.of(view), instrument);
+    return new AutoValue_MetricDescriptor(name, description, view, instrument);
   }
 
+  /**
+   * The name of the descriptor, equal to {@link View#getName()} if not null, else {@link
+   * InstrumentDescriptor#getName()}.
+   */
   public abstract String getName();
 
+  /**
+   * The description of the descriptor, equal to {@link View#getDescription()} if not null, else
+   * {@link InstrumentDescriptor#getDescription()}.
+   */
   public abstract String getDescription();
 
-  public abstract String getUnit();
+  /** The view that lead to the creation of this metric. */
+  public abstract View getSourceView();
 
-  /** The view that lead to the creation of this metric, if applicable. */
-  public abstract Optional<View> getSourceView();
   /** The instrument which lead to the creation of this metric. */
   public abstract InstrumentDescriptor getSourceInstrument();
 
+  /** The FQCN of the view aggregation. */
```

Comment:
How about using Class directly instead of String?

Reply:
Though otherwise if we already have the concept of a human-readable aggregation name for viewconfig, we should try to use that too